### PR TITLE
Support and glossary texts

### PR DESCRIPTION
**Pull request by eidarus**
### Master Redmine ticket
* (Required!)

### Child Redmine ticket(s)
* ()
* ()

### Local build result

```
bundle exec rake parallel:spec[4]
bundle exec cucumber
```

### Latest rebase/merge tag
*

---

### Data ticket(s) Followup
* (redmine links here - optional)

### Related Pull Requests
* (github links here - optional)

---

#### Peer Review
* (For code review)

#### Functional Testing
* (For testing locally)

#### Deployment
* (for release manager and/or build manager)

### CODE REVIEW

**Review by bill-transue — CHANGES_REQUESTED** [review]
It looks like out of pocket costs may be the only glossary item with a different term and title. Please double check the `FinancialAssistance::Benefit::INSURANCE_TYPE` one.

File: components/financial_assistance/app/views/financial_assistance/shared/_consumer_fields.html.erb [hunk 1]
```diff
@@ -82,7 +82,7 @@
   <div class="row row-form-wrapper no-buffer" id='indian_tribe_area'>
     <div class="row no-buffer">
       <div class="col-lg-6 col-md-6 col-sm-6 col-xs-12">
-        <label class="run-glossary">Are you a member of an American Indian or Alaska Native Tribe? *</label>
+      <label>Is this person a member of an <%= render partial:"shared/glossary", locals: {key: "american_indian", title: "American Indian", term: "American Indian" } %> or <%= render partial:"shared/glossary", locals: {key: "alaska_native", title: "Alaska Native", term: "Alaska Native" } %> Tribe? *</label>
```
[the remark this answers is not in the document]
Comment (ZASMan) [reply components/financial_assistance/app/views/financial_assistance/shared/_consumer_fields.html.erb:85]: Sometimes the title and term is the same?

File: components/financial_assistance/app/views/financial_assistance/deductions/index.html.erb [hunk 2]
```diff
@@ -61,8 +61,8 @@
               <%=check_box_tag "deduction_kind", deduction_kind, @applicant.deductions.of_kind(deduction_kind).present?, class: "deduction-checkbox-#{deduction_kind}" %>
             </div>
 
-            <div class="col-md-9 no-pd <%= deduction_kind %> run-glossary">
-              <%= FinancialAssistance::Deduction::DEDUCTION_TYPE[deduction_kind.to_sym] %>
+            <div class="col-md-9 no-pd <%= deduction_kind %>">
+              <%= render partial:"shared/glossary", locals: {key: "#{deduction_kind.to_sym}", title: "#{FinancialAssistance::Benefit::INSURANCE_TYPE[deduction_kind.to_sym]}", term: "#{FinancialAssistance::Benefit::INSURANCE_TYPE[deduction_kind.to_sym]}" } %>
```
[the remark this answers is not in the document]
Comment (eidarus) [reply components/financial_assistance/app/views/financial_assistance/deductions/index.html.erb:65]: `FinancialAssistance::Deduction::DEDUCTION_TYPE[deduction_kind.to_sym]` returns a full statement from components/financial_assistance/app/models/financial_assistance/deduction.rb, I just wanted the key it was using to pick which statement to return, and then I'm using that to grab the appropriate translation

[the remark this answers is not in the document]
Comment (bill-transue) [reply components/financial_assistance/app/views/financial_assistance/deductions/index.html.erb]: 👍🏼

**Review by bill-transue — APPROVED** [review]
Good work @eidarus, this will move the glossary to translations meaning they will be much easier to change and allow for client control.

**Pull request merged**